<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouter_entretien_mach</class>
 <widget class="QDialog" name="ajouter_entretien_mach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/sig/erp.jpg);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>750</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="iconnes.qrc">:/sig/images/erp.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>id_entretien</string>
   </property>
  </widget>
  <widget class="QComboBox" name="id_tech_CB">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QTextEdit" name="id_entretien_LE">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>278</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="entretiens_tableView">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>290</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>762</x>
     <y>278</y>
     <width>20</width>
     <height>441</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="actualise">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>actualis√©</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_ent">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2014</year>
     <month>1</month>
     <day>2</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>580</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Remarques</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>385</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>id_machine</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>338</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choisir la date d'entretien:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>1341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_Pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>178</y>
     <width>391</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>chef service maintenance</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tach_afaire_LE">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>198</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>ajouter les entretien des machines :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>taches a faire </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="remarques_LE">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>538</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="envoye_PB">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>648</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>envoyer au technicien</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>technicien responsable</string>
   </property>
  </widget>
  <widget class="QComboBox" name="id_mach_2_CB">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>378</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iconnes.qrc"/>
 </resources>
 <connections/>
</ui>
